app/models/user.rb

```diff
@@ -82,7 +90,19 @@ def password_reset_expired?
   end
 
   def feed
-    Micropost.latest.with_user id
+    Micropost.latest.with_user following_ids, id
```

Comment:
```suggestion
    Micropost.latest.with_user following_ids << id
```

ko cần 2 giá trị đâu. chỉ cần 1 array user_id thôi hì